### PR TITLE
Fix a gráfico

### PR DESCRIPTION
**Pull request by constanzamorales**
Se aumenta el mínimo de casos activos para que se vea mejor. Se logran mostrar todos los países en axis X.

### CODE REVIEW

**Review by NickGalaz — APPROVED** [review]
Ojo con JQUERY, debe utilizarse este script. <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
Debido a que a mi no me funciona de la otra fuente proporcionada. Me facilitaria para poder realizar las revisiones
